### ISSUE TITLE
Packages that have native dependency

### ISSUE THREAD

**Issue by m15a** [m15a/flake-awesome-neovim-plugins#127]
Some plugins have native dependencies. For example, [sniprun] depends on its Rust binary, and thus we should auto-update the `cargoHash`. 

This can be achieved using [nix-prefetch]:

```console
$ nix-prefetch '{ sha256 }: (import ./default.nix).packages.x86_64-linux.sniprun.bin.cargoDeps.overrideAttrs (_: { cargoSha256 = sha256; })'
The fetcher will be called as follows:
> <unnamed> {
>   sha256 = "sha256:0000000000000000000000000000000000000000000000000000";
> }

sha256-r0Uq8G84PAwxR9zCm3ox+82igGVphW7q8wLzYQrNdTk=
```

[sniprun]: https://github.com/michaelb/sniprun
[nix-prefetch]: https://github.com/msteen/nix-prefetch

**Comment by m15a** [comment]
Fixed a bug in aee4232a0d0d06991d7eee183092669a83d09f91

**Comment by m15a** [comment]
Also I noticed that nix-prefetch requires that any new rev hash is once stored in the JSON file. So, updating should be of two stage:

1. to update latest commit revs and tarball hashes, and then
2. to update extra hashes such as cargoSha256.
